Dshop/apps/products_catalogue/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.utils.text import slugify
+
+class CategoryBaseModel(models.Model):
```

Comment:
As we talked, please change to CatalogueBaseItem or similar.

Reply:
Ping :-)

Reply:
Done ;)